skyline-vscode/react-ui/src/Habitat.js

```diff
@@ -1,61 +1,55 @@
+import Subheader from "./Subheader";
+import { Container, Row, Spinner, Card } from "react-bootstrap";
+import BarGraph from "./components/HorizontalBarGraph";
 
-import Subheader from './Subheader';
+export default function Habitat({ habitatData }) {
+  // The colors used for the visualization
+  // The first n-1 colors should follow a blue gradient while the last
+  // color is used for the current device.
+  const colors = [
+    "#7986cb",
+    "#5c6bc0",
+    "#3f51b5",
+    "#3949ab",
+    "#303f9f",
+    "#283593",
+    "#1a237e",
+    "#ffc300",
+  ];
 
-import {
-    XAxis,
-    YAxis,
-    HorizontalBarSeries,
-    XYPlot,
-    makeWidthFlexible
-  } from 'react-vis';
+  let sampleData = [];
```

Comment:
Maybe we should rename this? I was confused by the variable name and initially thought it was mock data.

Reply:
Yeah, I changed to habitatProperties